packages/shared-ux/code_editor/impl/code_editor.tsx

```diff
@@ -695,3 +698,28 @@ const useBug175684OnChange = (onChange: CodeEditorProps['onChange']) => {
 
   return onChangeWrapper;
 };
+
+const useBug177756ReBroadcastMouseDown = () => {
+  const [$codeWrapper, setCodeWrapper] = React.useState<HTMLElement | null>(null);
+  const onCodeWrapperChange = React.useCallback(setCodeWrapper, [setCodeWrapper]);
```

Comment:
I thought that setState is stable and no need to wrap it in useCallback. might be wrong

Reply:
On second thought.. you're right, it definitely is an over kill.